static/app/components/devtoolbar/components/replay/replayPanel.tsx

```diff
@@ -0,0 +1,111 @@
+import {useContext} from 'react';
+import {css} from '@emotion/react';
+
+import {Button} from 'sentry/components/button';
+import AnalyticsProvider, {
+  AnalyticsContext,
+} from 'sentry/components/devtoolbar/components/analyticsProvider';
+import SentryAppLink from 'sentry/components/devtoolbar/components/sentryAppLink';
+import useReplayRecorder from 'sentry/components/devtoolbar/hooks/useReplayRecorder';
+import {resetFlexRowCss} from 'sentry/components/devtoolbar/styles/reset';
+import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {IconPause, IconPlay} from 'sentry/icons';
+import type {PlatformKey} from 'sentry/types/project';
+
+import useConfiguration from '../../hooks/useConfiguration';
+import {panelInsetContentCss, panelSectionCss} from '../../styles/panel';
+import {smallCss} from '../../styles/typography';
+import PanelLayout from '../panelLayout';
+
+const TRUNC_ID_LENGTH = 16;
+
+export default function ReplayPanel() {
+  const {projectSlug, projectId, projectPlatform, trackAnalytics} = useConfiguration();
+
+  const {disabledReason, isDisabled, isRecording, lastReplayId, start, stop} =
+    useReplayRecorder();
+
+  function ReplayLink({children}: {children: React.ReactNode}) {
+    const {eventName, eventKey} = useContext(AnalyticsContext);
+    return process.env.NODE_ENV === 'production' ? (
+      <SentryAppLink
+        to={{
+          url: `/replays/${lastReplayId}`,
+          query: {project: projectId},
+        }}
+      >
+        {children}
+      </SentryAppLink>
+    ) : (
+      <ExternalLink
+        href={`https://sentry-test.sentry.io/replays/${lastReplayId}/?project=5270453`}
```

Comment:
can you add links to the code that shows why this is needed? i suspect that we're conflating the env with the settings that relate to the env and the real if-statement should be inside [`useDevToolbar.tsx`](https://github.com/getsentry/sentry/blob/c9065b923e634ea79f788d4fce4f43d658590ae6/static/app/utils/useDevToolbar.tsx#L25-L32)

Reply:
> can you add links to the code that shows why this is needed?

Yeah, I don't have any atm but can take a look

> i suspect that we're conflating the env with the settings that relate to the env and the real if-statement should be inside [useDevToolbar.tsx](https://github.com/getsentry/sentry/blob/c9065b923e634ea79f788d4fce4f43d658590ae6/static/app/utils/useDevToolbar.tsx#L25-L32)

What do you mean?

Reply:
`useDevToolbar.tsx` hard-coded the project name and id's and stuff. If the DSN is configured then the orgSlug, projectSlug, etc should be updated to match

Reply:
Gotcha, I can move the `process.env.NODE_ENV === 'production'` check to useDevToolbar then and hard code the test org + project.

Also, to get the integration locally, I needed to comment out a line in getsentry's useReplayInit. We need to look at that file too, before sentry employees can use this